src/components/MultiHopTrade/components/TradeAmountInput.tsx

```diff
@@ -91,9 +111,17 @@ export type TradeAmountInputProps = {
 } & PropsWithChildren
 
 const defaultPercentOptions = [0.25, 0.5, 0.75, 1]
+const defaultFormValues = {
+  amountFieldInput: '',
+  amountCryptoPrecision: '',
+  amountUserCurrency: '',
+}
 
+// TODO: While this is called "TradeAmountInput", its parent TradeAssetInput is consumed by everything under the sun but swapper
+// Scrutinize this and rename all Trade references here, or at the very least in the parent to something more generic for sanity
```

Comment:
@woodenfurniture @0xApotheosis ^^ obviously not a now thing not to introduce regression risks but as this comment says we're super reliant on `<TradeAssetInput />` across domains and should probably do a follow-up to generalize things or at least make naming more generalized